numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -253,29 +257,16 @@ gentype_multiply(PyObject *m1, PyObject *m2)
      * Probably no class has both defined, but if they do, prefer number.
      */
     if (!PyArray_IsScalar(m1, Generic) &&
-            ((Py_TYPE(m1)->tp_as_sequence != NULL) &&
-             (Py_TYPE(m1)->tp_as_sequence->sq_repeat != NULL)) &&
-            ((Py_TYPE(m1)->tp_as_number == NULL) ||
-             (Py_TYPE(m1)->tp_as_number->nb_multiply == NULL))) {
-        /* Try to convert m2 to an int and try sequence repeat */
-        repeat = PyArray_PyIntAsIntp(m2);
```

Comment:
Looks like you need to get rid of the `repeat` declaration also.

Reply:
Good catch